acceptance/ui/features/api/cli_service.rb

```diff
@@ -148,6 +161,27 @@ def check_publicendpoint_port_exists_in_service?(service, endpoint, portAddr, pr
             return false
         end
 
+        # Looks up the given vhost from the vhosts table, and returns the enabled
+        # state based on the service definition.
+        def check_publicendpoint_vhost_enabled_in_service_json?(name)
```

Comment:
Thie code looks OK as is but it looks regular enough. Is it possible to have a common implementation somthing like check_publicpoint_enabled_in_service_json? with "port" and "vhost" as the parameter? It may simplify future maintenance. It applies to some other tests as well.